libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt

```diff
@@ -584,11 +584,10 @@ class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJa
     }
 
     @ReactMethod
-    fun buyBitcoin(provider: String, promise: Promise) {
+    fun buyBitcoin(req: BuyBitcoinRequest, promise: Promise) {
         executor.execute {
             try {
-                val buyBitcoinProvider = asBuyBitcoinProvider(provider)
-                val result = getBreezServices().buyBitcoin(buyBitcoinProvider)
+                val result = getBreezServices().buyBitcoin(req)
                 promise.resolve(result)
```

Comment:
This result needs to be mapped
```
promise.resolve(readableMapOf(result))
```

Reply:
Done in 3b9ef403befd5079c2e761d6dacd0f0beda0e125